packages/netlify-cms-backend-github/src/API.js

```diff
@@ -809,8 +832,8 @@ export default class API {
     );
   }
 
-  publishUnpublishedEntry(collection, slug) {
-    const contentKey = this.generateContentKey(collection.get('name'), slug);
+  publishUnpublishedEntry(collectionName, slug) {
+    const contentKey = this.generateContentKey(collectionName, slug);
```

Comment:
BUG: Looks like a bug introduced in https://github.com/netlify/netlify-cms/pull/2430

The first argument is actually a string and publishing an entry fails without this fix

Reply:
Fixed in #2523: https://github.com/netlify/netlify-cms/pull/2523/commits/9a63d11a9be15d0cd8c5a71141774b2175d9b284#diff-b4ba42c688a23ed9cec5139daec2c157L835-R836